### PR TITLE
Add mechanism to test typography variants with customers

### PR DESCRIPTION
**Pull request by keirog**
Provides a way for us to show different typography variants to customers.

The (hacky and inelegant) mechanism works via hash in the URL:
http://next.ft.com/[article-id]#typog-switcher:variant-[0-9]
This'll add a class to the body, to give a hook for varying CSS styles.
The variant is saved in localStorage to persist the styling for the customer.

Note: Only runs if the typographySwitcher flag is switched on, and obviously only works for enhanced-experience users.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "ft-poller": "0.0.4",
     "lodash": "^3.6.0",
     "merge": "^1.2.0",
-    "next-ft-api-client": "^3.0.0"
+    "next-ft-api-client": "^3.0.0",
+    "superstore": "^1.0.0"
```
Comment (matthew-andrews) [inline package.json]: lol whattt

Comment (keirog) [inline package.json]: Oh, bad? Seemed better than naked localStorage

Comment (matthew-andrews) [inline package.json]: haha no it is.  just a blast from the past.

**Comment by matthew-andrews** [comment]
:+1:

**Pull request merged**